src/log/LogClock.h

```diff
@@ -130,7 +134,7 @@ inline int append_time(const log_time& t, char *out, int outlen) {
   bool coarse = t.time_since_epoch().count().coarse;
   auto tv = log_clock::to_timeval(t);
   std::tm bdt;
-  localtime_r(&tv.tv_sec, &bdt);
+  localtime_r((time_t*)&tv.tv_sec, &bdt);
```

Comment:
this would be confusing to UNIX programmers i think. as tv_sec is known to be `time_t` see https://www.gnu.org/software/libc/manual/html_node/Time-Types.html. could you note in the commit message that this is to accommodate windows. see https://docs.microsoft.com/en-us/windows/win32/api/winsock/ns-winsock-timeval

Reply:
apart from this nit, LGTM.

Reply:
Makes sense, I've updated the commit message. Thanks for reviewing this PR!